.vitepress/config.ts

```diff
@@ -176,6 +176,45 @@ export default defineConfig({
             }
           ]
         }
+      ],
+      '/config/': [
+        {
+          text: '設定',
+          items: [
+            {　　　　　　　　　　
+              text: 'Vite の設定',
+              link: '/config/'
+            },
+            {
+              text: '共有オプション',
```

Comment:
各ページのタイトルに合わせてください:pray:
```suggestion
              text: '共通オプション',
```